src/core/directives/query/children_resolver.ts

```diff
@@ -323,7 +323,7 @@ export function _getParentCheckNotifiers( ctrl: DirectiveCtrl, requiredCtrls: Ob
               return false;
             }
             // check if current child is really queried from its parent
-            return ctrl instanceof propMetaInstance.selector;
+            return ctrl instanceof <Type>propMetaInstance.selector;
```

Comment:
I prefer `as` for type casting, makes things more clear and prevent's confusion with generics syntax